test/unit/files/actions/model.cds

```diff
@@ -85,3 +85,13 @@ service S2 {
     action a1 (p1: String, p2: many ExternalType2) returns ExternalType;
     action a2 (p1: String not null, p2: many ExternalType2, p3: Integer not null) returns ExternalType
 }
+
+entity Parent {}
```

Comment:
these new entities received no explicit test, but will cause an error during the smoke transpilation tests if the `override` modifier should go missing again.